src/subsonic.ts

```diff
@@ -1037,7 +1044,24 @@ export class Subsonic implements MusicService {
         .then(it => 
           it.find(station => station.id === id)!
         ),
-      
+      years: async () => {
+        const q: AlbumQuery = {
+          _index: 0,
+          _count: 100000,  // FIXME: better than this ?
```

Comment:
Is there a better way to do this ?

Reply:
probably, however it's good enough for now.  There is an initiative to improve the subsonic API, part of that addresses this type of thing so I'll tidy this up when I get a chance to look into the new API some more.